hil-test/tests/rmt.rs

```diff
@@ -80,6 +80,59 @@ fn generate_tx_data<const TX_LEN: usize>(write_end_marker: bool) -> [PulseCode;
     tx_data
 }
 
+// When running this with defmt:
+// - use `DEFMT_RTT_BUFFER_SIZE=32768 xtask run ...` to avoid truncated output
```

Comment:
This shouldn't be an issue 🤔

Reply:
At some point I had the impression that it helped, but in a quick test right now, things seems to be fine without (but see also below).